src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball();
```

Comment:
This method must contains a real implementation. But it returns always an empty Ball object. So why do we need this class at all?